### ISSUE TITLE
Option to uninstall choosenim

### ISSUE THREAD

**Issue by array-in-a-matrix** [dom96/choosenim#335]

**Comment by penguinite** [comment]
I assume you also want to uninstall the Nim toolchain that choosenim has installed. Since I don't think there is a way to keep Nim and to uninstall choosenim.

If you're on Linux, then simply run "rm -rf ~/.choosenim ~/.nimble" remove any lines in your shell configs that mention nimble or choosenim.

I have no idea how its like on Windows, but I imagine it's a similar process. Delete the folders, and then remove the environment variables.

If you want to uninstall *just* choosenim then, just run "rm ~/.nimble/bin/choosenim" on Linux. On windows, just navigate to .nimble/bin and delete choosenim.

**Comment by array-in-a-matrix** [comment]
Yeah I wanted to uninstall choosenim along with anything it installed as I wanted to switch to using the Nim package provided by the Arch Linux repositories. I did delete those directories but I wanted to make sure I didn't miss anything.

**Comment by penguinite** [comment]
Well then deleting both .nimble and .choosenim should get rid of everything as far as I am aware, since choosenim only installs in the user's home directory.
